testem.json

```diff
@@ -2,6 +2,7 @@
   "framework": "qunit",
   "test_page": "tests/index.html?hidepassed",
   "disable_watching": true,
+  "parallel": 4,
```

Comment:
This tells testem that it is allowed to use up to 4 containers in parallel. That's just an arbitrary number I set for testing between 2-4 containers.